rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +18,20 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+                    raise serializers.ValidationError(
+                        msg,
+                        code='authorization')
             else:
                 msg = _('Unable to log in with provided credentials.')
-                raise serializers.ValidationError(msg)
+                raise serializers.ValidationError(
```

Comment:
I guess these would be marginally cleaner reformatted as three lines...

    msg = ...
    code = ...
    raise serializers.ValidationError(msg, code)